src/main/java/com/hotels/bdp/circustrain/bigquery/copier/BigQueryCopierFactory.java

```diff
@@ -80,11 +46,7 @@ public Copier newInstance(
       List<Path> sourceSubLocations,
       Path replicaLocation,
       Map<String, Object> copierOptions) {
-    Copier copier = supportedFactory.newInstance(eventId, sourceBaseLocation, sourceSubLocations, replicaLocation,
-        copierOptions);
-    Copier bigQueryCopier = new BigQueryCopier(copier, dataExtractionManager);
-    log.info("Created copier which delegates to the copier produced by {}", supportedFactory.getClass().getName());
-    return bigQueryCopier;
+    return new BigQueryCopier(dataExtractionManager);
```

Comment:
This and the next method now look a bit odd in that we're ignoring most of the params that are passed in here.